src/app/components/home/clip/clip.component.ts

```diff
@@ -27,14 +27,13 @@ export class ClipComponent implements OnInit {
   @Input() largerFont: boolean;
   @Input() showMeta: boolean;
 
+  // numberOfThumbs = 0;
```

Comment:
This variable is never used so I commented it out; I suspect I can delete it safely -- or did you have some use for it in mind in the future? @cal2195

Reply:
Yeah, I think I was just duplicating other code and forgot about this! 👍